### ISSUE TITLE
Setting php.ini values in ruleset.xml

### ISSUE THREAD

**Issue by marczych** [squizlabs/PHP_CodeSniffer#560]
I'm putting all of my configuration into the [ruleset.xml file](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml) (it's actually `phpcs.xml` so the docs should probably be updated) which has been great so far. However, I can't seem to set `php.ini` values in there because the format is `-d key=value` but `<arg name="d" value="key=value" />` spits out an argument like `--d=key=value` which is invalid. I'm specifically trying to increase the memory limit because it doesn't work on my project otherwise.

Is there a way to set `php.ini` values from the xml file?

**Comment by gsherwood** [comment]
> Is there a way to set php.ini values from the xml file?

Not yet. I'll look at adding one using a custom tag.

> it's actually phpcs.xml so the docs should probably be updated

The name isn't important for ruleset files unless you are not actually telling PHPCS where to find the ruleset. In that case, it will look for one called phpcs.xml in the current directory. All ruleset files have exactly the same format.

**Comment by gsherwood** [comment]
I've added support for this now. It will be released in 2.5.0 (hopefully this week). You can set an ini setting using this format:
```xml
<ini name="name" value="value" />
```
Or, if just setting the value to TRUE:
```xml
<ini name="name" />
```
Proper example:
```xml
<ini name="memory_limit" value="64M" />
```
